rfc/text/wamp-terminology.md

```diff
@@ -0,0 +1,74 @@
+## Terminology {#terminology}
+
+This chapter collects the terminology *as used in WAMP*, the group of specialized words and respective meanings
+as used in this specification.
+
+Implementations SHOULD use terms as defined here in their public interfaces and documentation,
+and SHOULD NOT reinvent or reinterpret terms. Users SHOULD be able to transfer their knowledge gained on WAMP
+in the context of one implementation when starting to use another implementation.
+This is to support the overarching goal of WAMP to free application developers from restrictions when
+building distributed applications, both at the network level, and when choosing or switching the WAMP client
+libraries or routers used.
+
+Our goal is to maximize **User Choice and Experience** for developers of WAMP based applications,
+both formally (open protocol and open source) as well as practically (switching costs).
+
+
+### Fundamental
+
+{align="left"}
+| Term                              | Definition                                                                                            |
+|-----------------------------------|-------------------------------------------------------------------------------------------------------|
+| *User*                            | A user runs *Client*s or *Router*s which implement WAMP                                               |
+| *Client*                          | A program run by a *User* with application code using WAMP for application level communication        |
+| *Router*                          | A program run by a *User* with middleware code using WAMP to provide application routing services     |
+| *Peer*                            | A WAMP *Client* or *Router*. An implementation might embed and use both roles                         |
+| *Realm*                           | Isolated WAMP URI namespace, routing and administrative domain, optionally protected by **AA**        |
+| *Transport*                       | The bidirectional, ordered, full duplex message channel over which *Peers* communicate                |
+| *Connection*                      | When a *Client* is using a *Transport* over a network, the underlying network connection              |
```

Comment:
The connection can be made locally, via unix socket, interprocess RPC, or even internally inside one running program. So I think it should be not bound to the network but rather focus on the logical communication channel between peers.